docs/reference/search/rank-eval.asciidoc

```diff
@@ -135,11 +142,12 @@ GET /my_index/_rank_eval
 -----------------------------
 // NOTCONSOLE
 
-<1> the search request's id, used to group result details later 
-<2> the query that is being evaluated
-<3> a list of document ratings, each entry containing the document's `_index` and 
-`_id` together with the rating of the document's relevance with regard to this 
-search request
+<1> The search request's ID, used to group result details later.
+<2> The query being evaluated.
+<3> A list of document ratings. Each entry contains the following arguments:
+- `_index`: The document's data stream or index
```

Comment:
Unfortunately, the `_index` entry must refer to either the index or specific backing index of a data stream. The data stream itself cannot be specified there.